Tweak/Core/Views/KayokoView.m

```diff
@@ -197,7 +197,7 @@ - (void)handleHeaderButtonPressed {
 
     if ([[self historyTableView] isHidden]) {
         NSArray* items = [[PasteboardManager sharedInstance] itemsFromHistoryWithKey:kHistoryKeyHistory];
-        [[self favoritesTableView] reloadDataWithItems:items];
+        [[self historyTableView] reloadDataWithItems:items];
```

Comment:
Bugfix: when switch back from favorites view, items in history view will disappear.